src/coreclr/jit/gentree.cpp

```diff
@@ -1344,41 +1344,6 @@ bool CallArg::IsUserArg() const
     }
 }
 
-#ifdef DEBUG
-//---------------------------------------------------------------
-// CheckIsStruct: Verify that the struct ABI information is consistent with the IR node.
-//
-void CallArg::CheckIsStruct()
```

Comment:
I decided to remove this; the backend is already be doing similar validation for IR shapes depending on the ABI information.